pycrostates/cluster/tests/test_kmeans.py

```diff
@@ -293,6 +293,12 @@ def test_reorder(caplog):
     assert ModK.clusters_names[0] == ModK_.clusters_names[1]
     assert ModK.clusters_names[0] == ModK_.clusters_names[1]
 
+    # test .labels_ reordering
+    x = ModK_.labels_[:20]
+    # x = [3, 1, 1, 2, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0]
+    y = [3, 2, 2, 1, 2, 2, 2, 2, 2, 2, 2, 2, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
So following my comment above that order is 0-indexed, `ModK_.reorder_clusters(order=np.array([1, 0, 2, 3]))` should:
- Place cluster 1 in position 0
- Place cluster 0 in position 1
- Keep cluster 2 in position 2
- Keep cluster 3 in position 3

Thus expected labels should be: `[3, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1]`.